### ISSUE TITLE
no logging at all

### ISSUE THREAD

**Issue by mbande** [Parsely/streamparse#331]
i've configured logging as this in config.json:
```
            "log": {
                "path": "/var/log/storm",
                "file": "pystorm_{topology_name}_{component_name}_{task_id}_{pid}.log",
                "max_bytes": 1000000,
                "backup_count": 10,
                "level": "info"
            },
```
the the log files are created for each component/task/pid. however almost every bolt has statements like ```self.logger.info('blah blah')``` in their process method, there is no log entry in files after many tuple processing

**Comment by Darkless012** [comment]
Try to use self.logger.warn() or self.logger.error() to see if the problem is with "log_level" configuration or somewhere else.

**Comment by mbande** [comment]
already tried with self.logger.warn() with no effect

**Comment by mbande** [comment]
even ```sparse run``` does not print log messages in stdout
is it normal?

**Comment by dan-blanchard** [comment]
@mbande No, that's very strange. What versions of streamparse and pystorm are you using?

**Comment by mbande** [comment]
streamparse==3.2.0
pystorm==3.0.3

**Comment by mbande** [comment]
just found the root cause.
the topology is used to run components in a Django project context
in Django settings, there is a ```disable_existing_loggers``` flag that expresses itself!
and that beats the streamparse logging configurations
